hca/cli.py

```diff
@@ -107,12 +107,14 @@ def help(args):
 
 
 def main(args=None):
-    if '--help' in sys.argv or '-h' in sys.argv:
+    if not args:
+        args = sys.argv[1:]
```

Comment:
@DailyDreaming i had to do this to get around parser issue when trying to use the cli after performing a `make install` everything works within testing + manual cli usage. Can you think of a reason it would break a parser later on?

Reply:
No, I think that was the correct call.